src/olympia/migrations/952-correct-index-theme-user-counts.sql

```diff
@@ -0,0 +1,5 @@
+-- Drop old index first
+DROP INDEX `addon_date_idx` ON `theme_user_counts`;
+
+-- Now create a proper
+ALTER TABLE `theme_user_counts` ADD CONSTRAINT `theme_user_counts_date_cc9034dde90789f_uniq` UNIQUE (`date`, `addon_id`);
```

Comment:
OTOH if I create the django migration, `sqlmigrate` does result in `ALTER TABLE `theme_user_counts` ADD CONSTRAINT `theme_user_counts_date_cc9034dde90789f_uniq` UNIQUE (`date`, `addon_id`);` so it looks ok, ignore my previous comment.